src/blocks/CardLayout/CardLayout.tsx

```diff
@@ -29,19 +29,32 @@ const CardLayout: React.FC<CardLayoutBlockProps> = ({
     children,
     className,
     titleClassName,
-}) => (
-    <AnimateBlock className={b(null, className)} animate={animated}>
-        {(title || description) && (
-            <Title title={title} subtitle={description} className={titleClassName} />
-        )}
-        <Row>
-            {React.Children.map(children, (child, index) => (
-                <Col key={index} sizes={colSizes} className={b('item')}>
-                    {child}
-                </Col>
-            ))}
-        </Row>
-    </AnimateBlock>
-);
+    background,
+}) => {
+    const withBackground = Boolean(
```

Comment:
why not just `const withBackground = Boolean(background)` ?

Reply:
Theoretically it is possible to pass an empty object to `background` prop which would do nothing inside the `BackgroundImage` rendering but it adds additional indents. 
I added the same logic in the `src/blocks/Form/Form.tsx` as well.
Should I fix it also?